client/rpcserver/handlers.go

```diff
@@ -475,64 +536,59 @@ Registration is complete after the fee transaction has been confirmed.`,
 	exchangesRoute: {
 		pwArgsShort: ``,
 		argsShort:   ``,
-		cmdSummary:  `List assets supported by all registered DEX servers their and active trades.`,
+		cmdSummary:  `Detailed information about known exchanges, markets, and active trades.`,
 		pwArgsLong:  ``,
 		argsLong:    ``,
 		returns: `Returns:
     map: The exchanges result.
     {
-      "[DEX url] (map): {
-        "url" (string): The DEX URL.
-        "markets" (map): {
-          "[assetID-assetID]" (map): {
-            "name" (string): The market name. e.g. "dcr_btc".
-            "baseid" (int): The base assetID
-            "basesymbol" (string): The base assetID symbol.
-            "quoteid" (int): The quote assetID.
-            "quotesymbol" (string): The quote assetID symbol,
-            "epochlen" (int): Duration of a trade window in milliseconds.
-            "startepoch" (int): Unix time of start of the last epoch in seconds
+      "[DEX url]": {
+        "markets": {
```

Comment:
The redundant "url" is removed here, but it's still in the test data https://github.com/decred/dcrdex/pull/314/commits/5255b3040272b8cf5bad78d95355cbd17ee0512d#diff-d90335238d9d82969d7f80519eb6297eR448

Reply:
That is supposed to be the input from the core method. The field is gone from the output. I hope the new test is better and easier to understand.

Reply:
Ah, my bad. Thanks.